python/paddle/fluid/tests/unittests/test_center_loss.py

```diff
@@ -0,0 +1,78 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+from op_test import OpTest
+import paddle.fluid.core as core
+
+
+class TestCenterLossOp(OpTest):
+    def setUp(self):
+        self.op_type = "center_loss"
+        self.dtype = np.float32
+        self.init_dtype_type()
+        batch_size = 6
+        feet_dim = 10
+        cluster_num = 8
+        self.attrs = {}
+        self.attrs['cluster_num'] = cluster_num
+        self.attrs['lambda'] = 0.1
+        self.attrs['need_update'] = True
```

Comment:
missing unittest for `need_update` is False, which is an import feature

Reply:
done